terraform/environments/mlra/modules/cloudwatch-alarms/variables.tf

```diff
@@ -0,0 +1,124 @@
+variable "alarmname" {
+  type    = string
```

Comment:
Just an FYI - type = string is implied if type is not declared :)

Reply:
noted